src/main/java/by/onliner/core/properties/EnvironmentConfig.java

```diff
@@ -0,0 +1,48 @@
+package by.onliner.core.properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Properties;
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String HOST = "host";
+    private static final Properties properties;
+
+    private final static String ENVIRONMENT_PROPERTIES_FILENAME = String.format("application-%s.properties", Environment.getEnvironment().getName());
```

Comment:
Why is this variable global? are you going to use it somewhere else?

Reply:
Fixed